Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -1212,20 +1212,22 @@ static float currentBacklightLevel()
 }
 #endif
 
-void WebProcess::grantAccessToAssetServices(WebKit::SandboxExtension::Handle&& mobileAssetV2Handle)
+void WebProcess::grantAccessToAssetServices(Vector<WebKit::SandboxExtension::Handle>&& assetServicesHandles)
 {
-    if (m_assetServiceV2Extension)
+    if (m_assetServicesExtensions.size() > 0)
```

Comment:
It's more idiomatic to say the more terse "if (m_assetServicesExtensions.size())".

Reply:
Good catch, will fix.

Thanks for reviewing!